### DISCUSSION TITLE
Initial setup - unable to login

### DISCUSSION THREAD

**Discussion by khalavak** [intelowlproject/IntelOwl#1935] [General]
Hello!
New user and just started Intelowl on a local instance to test it out. Configured the basics in the env files and created a superuser account. I disabled the RECAPTCHA as I don't need that: 

docker/env_file_app:USE_RECAPTCHA=False
docker/env_file_app:RECAPTCHA_SECRET_KEY="localhost"

When I now try to login the login page it says "ERROR for site owner: Invalid site key" and the login button can not be clicked to login...any ideas what's wrong?

**Comment by mlodic** [comment]
mmm quite strange. Could you please restore the default values for those variables and re-start the application?

**Comment by khalavak** [comment]
OK will try...might have misunderstood...after creating the first user with this command, can I use that account to login to Intelowl at /login page or do I have to go into Django Admin and create the actual users that will login to the IntelOwl UI?

`docker exec -ti intelowl_uwsgi python3 manage.py createsuperuser`

If I try to access the /admin/ page I get the following error: 

```
{
"code": 503,
"message": "Service Temporarily Unavailable"
}
```

**Comment by mlodic** [comment]
you should first create the Django Admin superuser and then create the normal users from there.

Otherwise, you could leverage the Google Oauth2 integration and login with Google accounts. That would save you some time. Check the docs for the details

**Comment by khalavak** [comment]
OK thanks. Is it possible to use other Oauth2 providers, like Keycloak? If not using Google and for example, Microsoft AzureAD, is that possible also?

**Comment by mlodic** [comment]
Nope, that would require custom development. If you need that and you have some python developers, you could easily add such integrations in the framework. The documentation is easy to follow and we can help in the process.
